### PR TITLE
Document how to deploy cluster with BM and VM workers

### PR DESCRIPTION
**Pull request by adduarte**

### CODE REVIEW

File: docs/user/openstack/deploy_baremetal_workers.md [hunk 1]
```diff
@@ -177,6 +178,44 @@ Plane and Compute nodes. The cluster is deployed to a preexisting network.
     If the installer times out waiting for the bare-metal workers to complete booting,
     restart the installation using [the appropriate *wait-for* command](#considerations-when-deploying-bare-metal-workers).
 
+deploying-cluster-with-mixture-of-vm-and-bm-workers
```
Comment (EmilienM) [inline docs/user/openstack/deploy_baremetal_workers.md:181]: I don't think you need that line.

**Review by EmilienM — CHANGES_REQUESTED** [review]
One leftover, see inline

File: docs/user/openstack/deploy_baremetal_workers.md (hunk 1 above)
Comment (adduarte) [reply docs/user/openstack/deploy_baremetal_workers.md:181]: no I don't

**Comment by pierreprinetti** [comment]
/approve
/lgtm
:+1:

**Comment by pierreprinetti** [comment]
/hold
Please add a reference to a BZ or Jira card so that the change doesn't escape QE and dowstream-docs

**Pull request merged**